### DISCUSSION TITLE
Shiny for Python with Quarto

### DISCUSSION THREAD

**Discussion by tomkom** [quarto-dev/quarto-cli#1694] [Q&A]
I am keen to bring some interactivity to my websites generated via quarto, containing Python code.
I am excited by the Shiny for Python ( as I am hoping it will be a better supported solution than ipywidgets etc).

Is there any example of using quarto with Shiny for Py yet [link](https://shiny.rstudio.com/py/)?

**Comment by jjallaire** [comment]
Here is one example of using Shiny for Python w/ WASM (so 100% client side): https://github.com/wch/pyshiny-quarto-demo

We'll be making this into a template that has some better documentation soon but that may get you going in the meantime.

**Comment by psychemedia** [comment]
I note the release of `Shinylive`: _"embed [Shinylive](https://shiny.rstudio.com/py/docs/shinylive.html) applications in a Quarto document. These are [Shiny for Python](https://shiny.rstudio.com/py/) applications which run completely in the browser, using [Pyodide](https://pyodide.org/) (Python compiled to WebAssembly)."_

See also the [announcement post](https://quarto.org/docs/blog/posts/2022-10-25-shinylive-extension/).
